src/services/aws/aws.ts

```diff
@@ -113,7 +113,7 @@ export const aws = createApi({
       providesTags: ["profile"],
       query: ({ endowId, isLegacy = false }) => ({
         params: { legacy: isLegacy },
-        url: `/${v(2)}/profile/${network}/endowment/${endowId}`,
+        url: `/${v(1)}/profile/endowment/${endowId}`,
```

Comment:
See Paolo's comment https://linear.app/angel-protocol/issue/AP-846/new-balances-and-metrics-in-profile-and-admin-pages#comment-5adf91f7